Makefile

```diff
@@ -172,6 +177,55 @@ docker-push:
 	docker push ${DELETION_DEFENDER_IMG}
 	docker push ${UNMANAGED_DETECTOR_IMG}
 
+__tooling-image:
+	docker buildx build build/tooling \
+		--platform="$(PLATFORM)" \
+		--output=$(OUTPUT_TYPE) \
+		-t kcc-tooling
+
+__controller-gen: __tooling-image
+CONTROLLER_GEN=docker run --rm -v $(shell pwd):/wkdir kcc-tooling controller-gen
+
+.PHONY: rename-crds
+rename-crds:
+	@echo "Renaming generated CRDs..."
+	@cd config/crds/tmp && \
+	for file in *.yaml; do \
+		if [ "$$file" != "kustomization.yaml" ]; then \
+			base_name=$$(echo "$$file" | sed 's/apiextensions.k8s.io_v1_customresourcedefinition_//; s/.yaml//'); \
+			domain=$$(echo "$$base_name" | cut -d'_' -f1); \
+			resource=$$(echo "$$base_name" | cut -d'_' -f2-); \
+			new_name="apiextensions.k8s.io_v1_customresourcedefinition_$${resource}.$${domain}.yaml"; \
+			mv "$$file" "$$new_name"; \
+			echo "Renamed $$file to $$new_name"; \
```

Comment:
Yeah, I _guess_ we care about matching the existing names :-)  I don't think anything breaks, but it seems good in terms of risk reduction.

Reply:
More of a nice to have (for me), agreed.